### DISCUSSION TITLE
关于傀影肉鸽重招募/部署逻辑的疑问？

### DISCUSSION THREAD

**Discussion by kongwei981126** [MaaAssistantArknights/MaaAssistantArknights#5139] [Q&A] [answered]
今天在自动傀影肉鸽，然后发现resource/roguelike里的recruitment，copilot和shopping似乎只包含关于水月肉鸽的战斗部署顺序/招募/购买的信息，傀影肉鸽的大部分关卡只有标注红蓝门（堵门位置和方向）。请问傀影肉鸽是没有类似的支持吗？

（代码水平不太好翻了半天也没太看懂）如果我想自定义傀影肉鸽的招募/部署逻辑，在对应的json里效仿水月肉鸽的写法可以正常运行吗？

**Comment by ABA2396** [comment, marked as answer]
可以的，傀影那时候只有红蓝门和自动部署的逻辑，水月出来之后才支持部署逻辑，也就没补傀影的了
